Update Docs ping in issue template (#1061)
github/vscode-codeql: .github/pull_request_template.md

- [ ] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [ ] Issues have been created for any UI or other user-facing changes made by this pull request.

> - [ ] The `ready-for-doc-review` label has been added to all issues for UI or other user-facing changes made by this pull request.

Reviewer: ```suggestion
- [ ] If this pull request makes user-facing changes that require documentation changes, the `ready-for-doc-review` label has been added to this pull request or the corresponding issue.
```

Author: Good call! Thanks, Aditya. ✨
